contextualSpellCheck/contextualSpellCheck.py

```diff
@@ -73,121 +119,134 @@ def __init__(self, vocab_path="", debug=False, performance=False):
                 words = []
 
         if vocab_path == "":
-            currentPath = os.path.dirname(__file__)
-            vocab_path = os.path.join(currentPath, "data/vocab.txt")
+            current_path = os.path.dirname(__file__)
+            vocab_path = os.path.join(current_path, "data/vocab.txt")
             with open(vocab_path, encoding="utf8") as f:
                 # if want to remove '[unusedXX]' from vocab
-                # words = [line.rstrip() for line in f if not line.startswith('[unused')]
+                # words = [
+                #     line.rstrip()
+                #     for line in f
+                #     if not line.startswith("[unused")
+                # ]
                 words = [line.strip() for line in f]
 
+        self.max_edit_dist = max_edit_dist
+        self.model_name = model_name
         self.vocab = Vocab(strings=words)
-        self.BertTokenizer = AutoTokenizer.from_pretrained("bert-base-cased")
-        self.BertModel = AutoModelWithLMHead.from_pretrained("bert-base-cased")
+        self.BertTokenizer = AutoTokenizer.from_pretrained(self.model_name)
+        self.BertModel = AutoModelForMaskedLM.from_pretrained(self.model_name)
         self.mask = self.BertTokenizer.mask_token
         self.debug = debug
         self.performance = performance
         if not Doc.has_extension("contextual_spellCheck"):
             Doc.set_extension("contextual_spellCheck", default=True)
             Doc.set_extension("performed_spellCheck", default=False)
 
-            # {originalToken-1:[suggestedToken-1,suggestedToken-2,..],
-            #  originalToken-2:[...]}
             Doc.set_extension("suggestions_spellCheck", default={})
             Doc.set_extension("outcome_spellCheck", default="")
             Doc.set_extension("score_spellCheck", default=None)
 
             Span.set_extension(
-                "get_has_spellCheck", getter=self.span_require_spellCheck
+                "get_has_spellCheck", getter=self.span_require_spell_check
```

Comment:
I thought about snake_case but I think as the name of the library is in CamelCase (**contextualSpellCheck**), I will like to stick with camelCase at least with "spellCheck". 
```suggestion
                "get_has_spellCheck", getter=self.spanRequireSpellCheck
```

or use function name styling with underscore:
```suggestion
                "get_has_spellCheck", getter=self.span_require_spellCheck
```
What I think is `spellCheck` should be left as is. I dont mind changing stles for others!

Reply:
In my opinion, we use CamelCase  for class but for function snake_case is best.

Reply:
I also read somewhere what in python snake_case preferred. Let go with that then, CamelCase for class and snake_case for function.